### DISCUSSION TITLE
Error rendering in blade using Livewire & Alpine

### DISCUSSION THREAD

**Discussion by Anglin-George** [livewire/livewire#5023] [1. Help] [answered]
Is it possible to bind index instead of `0` below by alpine ?

`
@error('array.0.item')
 <span class="invalid-feedback" role="alert">
   <strong>{{ $message }}</strong>
 </span>
@enderror
`
---------------------------------------------------------------------------------------
Below I'm trying to explain my current situation.

I have a dynamic table in my form. Whenever I click to add row it appends a same set of input fields to table. This table inside of a <template> which is rendered by x-for loop.

Sample : 
-------------------------------------------------------------
array item text box 1   |     array item text box 2    |     Delete Button
-------------------------------------------------------------
array item text box 1   |     array item text box 2    |     Delete Button
-------------------------------------------------------------
.
.

The add section done by alpine js. server validation get from livewire. The validation working successfully.  I get the errors with its index number. 
like : array.0.item, array.1.item, ..
I can show the error if I'm hard coded as

@error('array.0.item')
 <span class="invalid-feedback" role="alert">
   <strong>{{ $message }}</strong>
 </span>
@enderror

But for this dynamic situation would not useful for this solution.

-------------------------------------------------------------------------------------------

So any help much appreciated.

![Screenshot 2022-06-09 185150](https://user-images.githubusercontent.com/30459349/172857261-9a697ef5-c04a-454b-8a41-f99ed4aa2448.png)
![Screenshot 2022-06-09 185252](https://user-images.githubusercontent.com/30459349/172857472-bad8bdcb-fd32-4892-8de8-ce28b87614fd.png)

**Comment by joshhanley** [comment, marked as answer]
@Anglin-George Livewire won't work inside an Alpine loop, but you can use a blade foreach loop instead to render the dynamic rows.

Hope this helps!

**Reply by Anglin-George** [reply]
Thanks for the reply @joshhanley. 

For now I just use * notation. So when any input not entered in required field it shows an error ( But even in the entered field also 😢). I changed error text to fill all the required field.

-------------------------------------------

But I will definitely try this one. Thanks in advance.

**Reply by joshhanley** [reply]
@Anglin-George no worries! 🙂
